tests/func/run/subnet.yml

```diff
@@ -0,0 +1,21 @@
+# FIXME: Find a way we can test the whole playbook rather than just
+# the role. Either make sure we're not in a play here and use
+# import_playbook, or spawn an ansible-playbook subprocess? The latter
+# might be actually a more precise way to test the real end-user
+# experience.
+- include_role:
+    name: os_migrate.os_migrate.export_subnets
+  vars:
+    export_subnets_name_filter:
+      - regex: '^osm_'
+
+- name: load exported data
+  set_fact:
+    resources: "{{ (lookup('file', os_migrate_data_dir + '/subnets.yml') | from_yaml)
```

Comment:
I think the tests fail because the "resources" name here clashes with the one in network test, they overwrite each other. Sometimes overwriting doesn't matter and saves memory, but in this case we reuse the fact in idempotency testing, so we will have to start namespacing here. `network_resources` and `subnet_resources`

Reply:
thanks.  changing the resources to subnet_resources made the test pass.